### PR TITLE
dm/show-bin

### PR DESCRIPTION
**Pull request by dmerejkowsky**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# Next release
+
+* Add `dmenv show:bin_path` to show path of the virtualenv's binaries.
```
Comment (tux3) [inline CHANGELOG.md]: Nit: s/show path/show the path/

**Comment by tux3** [comment]
Not a huge deal, but it would have been nice to split up the unrelated doc updates into a different commit than "add a show:bin_path command"...

File: src/lib.rs [hunk 2]
```diff
@@ -79,6 +79,7 @@ pub fn run(cmd: Command) -> Result<(), Error> {
         }
         SubCommand::ShowDeps {} => venv_manager.show_deps(),
         SubCommand::ShowVenvPath {} => venv_manager.show_venv_path(),
+        SubCommand::ShowVenvBin {} => venv_manager.show_venv_bin(),
```
Comment (tux3) [inline src/lib.rs]: Maybe `ShowVenvBinPath` and `show_venv_bin_path` ?

That'd be consistent with the `get_venv_bin_path(&self)` you have below

**Comment by dmerejkowsky** [comment]
> uge deal, but it would have been nice to split up the couple unrelated doc updates into a different commit than "add a show:bin_path command"

.... and that's how you discourage people from writing documentation :P

**Comment by tux3** [comment]
> > it would have been nice to split up the couple unrelated doc updates into a different commit than "add a show:bin_path command"
> 
> .... and that's how you discourage people from writing documentation :P

Aw, in my defense I did say it's not a big deal :sweat_smile: 

Let me explain why I'm being annoying: I actually think it's cool if you don't want to bother splitting things up super finely, it really is. 

But in a single command you could rename the commit to something like "Add show:bin_path and update docs". I just like when the commit message reflects that, so I'm not caught off-guard!

**Pull request merged**